carma_wm/include/carma_wm/WorldModel.h

```diff
@@ -144,6 +144,12 @@ class WorldModel
    */
   virtual LaneletRoutingGraphConstPtr getMapRoutingGraph() const = 0;
 
+  /*! \brief Get most recent roadway objects - all objects on the road detected by the camera fusion.
```

Comment:
change "detected by the camera fusion" to "detected by perception stack"
It may or may not involve a camera depending on the sensor set.